CHANGELOG.md

```diff
@@ -1,6 +1,201 @@
 Change Log
 ==========
 
+Version 3.0.0-alpha01 *(2019-03-14)*
+------------------------------------
+
+ * New: Kotlin Generator
+ 
+   Wire 3 can generate Kotlin data classes. To enable this feature via the command line API, pass in 
+   the `--kotlin_out` parameter that should specify the output directory for the generated `*.kt` 
+   files. 
+   
+   Given the following simple proto:
+   
+   ```proto
```

Comment:
wanna get even more aggressive about brevity?

```
   message Person {
     required string name = 1;
     required int32 id = 2;
   }
   ```

   the generated Kotlin code will look like the following:

   ```kotlin
   data class Person(
     @field:WireField(tag = 1, adapter = "com.squareup.wire.ProtoAdapter#STRING") 
     val name: String,
     @field:WireField(tag = 2, adapter = "com.squareup.wire.ProtoAdapter#INT32")
     val id: Int
   ) : Message<Person, Person.Builder>(ADAPTER, unknownFields) {
     companion object {
       @JvmField
       val ADAPTER: ProtoAdapter<Person> = ... // code omitted for brevity
     }
   }
```

Then you could say that the builder can be added rather than subtracted?